packages/framework/src/Migrations/Version20240828202437.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20240828202437 extends AbstractMigration
```

Comment:
this commit will be dropped after the structure is updated in the "base" user story, right?

Reply:
commit removed